src/java.desktop/share/classes/java/awt/MediaTracker.java

```diff
@@ -84,28 +88,33 @@
  *      // Get the images for the background (id == 0)
  *      // and the animation frames (id == 1)
  *      // and add them to the MediaTracker
- *      public void init() {
+ *      public static void main(String[] args) throws Exception {
+ *          MediaTrackerExample mte = new MediaTrackerExample();
+ *          EventQueue.invokeAndWait(() -> {
+ *              Frame frame = new Frame("MediaTrackerExample");
+ *              frame.setSize(400, 400);
+ *              frame.add(mte);
+ *              frame.setVisible(true);
+ *          });
+ *          mte.startAnimation();
+ *      }
+ *
+ *      public MediaTrackerExample() {
+ *          Toolkit tk = Toolkit.getDefaultToolkit();
  *          tracker = new MediaTracker(this);
- *          bg = getImage(getDocumentBase(),
- *                  "images/background.gif");
+ *          // Note : actual images not provided as part of this code example
+ *          bg = tk.getImage("background.gif");
  *          tracker.addImage(bg, 0);
  *          for (int i = 0; i < 5; i++) {
- *              anim[i] = getImage(getDocumentBase(),
- *                      "images/anim"+i+".gif");
+ *              anim[i] = tk.getImage("anim" + i + ".gif");
  *              tracker.addImage(anim[i], 1);
  *          }
  *      }
  *
- *      // Start the animation thread.
- *      public void start() {
+ *      public void startAnimation() {
  *          animator = new Thread(this);
  *          animator.start();
- *      }
- *
- *      // Stop the animation thread.
- *      public void stop() {
- *          animator = null;
- *      }
+ *     }
```

Comment:
```suggestion
 *      }
```
Correct the indentation.